node/sync.go

```diff
@@ -157,12 +159,39 @@ func (d *Pegnetd) SyncBlock(ctx context.Context, tx *sql.Tx, height uint32) erro
 				return err
 			}
 		}
+
+		if err := d.PayWinners(tx, winners); err != nil {
+			return err
+		}
+
 	}
 
 	// TODO: Handle converts/txs
 	return nil
 }
 
+func (d *Pegnetd) PayWinners(tx *sql.Tx, winners []*grader.GradingOPR) error {
+	// Reward the winners
+	for i := range winners {
+		addr, err := factom.NewFAAddress(winners[i].OPR.GetAddress())
+		if err != nil {
+			// TODO: This is kinda an odd case. I think we should just drop the rewards
```

Comment:
I agree with this. An invalid address should just not get paid. If their putting enough resources into mining, its on them to type their address right.

Reply:
this will be addressed by https://github.com/pegnet/pegnet/pull/326, oprs with invalid addresses will get discarded by the grader